ethers-signers/src/ledger/app.rs

```diff
@@ -204,25 +225,47 @@ impl LedgerEthereum {
             response_len: None,
         };
 
-        let mut result = Vec::new();
+        let mut answer = None;
+        // workaround for https://github.com/LedgerHQ/app-ethereum/issues/409
+        // TODO: remove in future version
+        let chunk_size =
+            (0..=255).rev().find(|i| payload.len() % i != 3).expect("true for any length");
```

Comment:
finds the largest chunk size that does not leave 3 hanging bytes on the payload lol

Reply:
ridiculous